### PR TITLE
A smattering of commits to get the build / deploy flow working again

### PR DESCRIPTION
**Pull request by ianconsolata**
This is a hodgepodge of updates and changes I needed to make in order to get the project building again. None of them had major substantive changes, except that I had to disable minification. The non-minified build works fine, however, so for now just use the non-minified version so we can start making updates again.

### CODE REVIEW

**Comment by ianconsolata** [comment]
@cwaring got things building in Fleek again. Can you review and merge this so I can start clearing out the backlog of other PRs?

**Review by johnnymatthews — APPROVED** [review]
Nice job clearing this repo up @ianconsolata! It's been borked since 2021!

File: .fleek.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "build": {
-    "image": "hugomrdias/puppeteer",
+    "image": "node:20.2",
```
Comment (cwaring) [inline .fleek.json:3]:
```suggestion
    "image": "node:20-slim",
```

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v20.2.0
```
Comment (cwaring) [inline .nvmrc:1]:
```suggestion
v20
```

**Review by cwaring — APPROVED** [review]
nice refactor! switching the base image to a smaller v20 distro but nothing blocking for now. great to have this working again 🚢

**Pull request merged**